### PR TITLE
fix(clerk-js): Fix clipped phone input in <UserProfile/>

### PR DESCRIPTION
**Pull request by octoper**
## Description

This PR fixes an issue we have with Phone input in `<UserProfile/>`

## Before:
CleanShot 2024-02-05 at 18 22 17@2x

## After:
CleanShot 2024-02-05 at 18 21 51@2x

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by octoper** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @octoper, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-2at6fja6v.clerkstage.dev) | Feb 05, 2024 04:26 PM |

File: packages/clerk-js/src/ui/elements/Action/ActionCard.tsx [hunk 1]
```diff
@@ -26,7 +26,6 @@ export const ActionCard = (props: ActionCardProps) => {
           boxShadow: t.shadows.$actionCardShadow,
           gap: t.space.$4,
           borderRadius: t.radii.$lg,
-          overflow: 'hidden',
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/elements/Action/ActionCard.tsx:29]: Seems logical, @desiprisg do you remember why we added this in the first place ?

**Review by anagstef — APPROVED** [review]

File: packages/clerk-js/src/ui/elements/Action/ActionCard.tsx (hunk 1 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/Action/ActionCard.tsx:29]: Can't think of anything right now. Nothing seems broken. Makes sense to remove this.

**Review by desiprisg — APPROVED** [review]

**Pull request merged**